diff --git a/node_modules/recharts/es6/cartesian/CartesianAxis.js b/node_modules/recharts/es6/cartesian/CartesianAxis.js
index 7aafddf..a68b75e 100644
--- a/node_modules/recharts/es6/cartesian/CartesianAxis.js
+++ b/node_modules/recharts/es6/cartesian/CartesianAxis.js
@@ -255,15 +255,12 @@ export var CartesianAxis = /*#__PURE__*/function (_Component) {
           tickCoord = _this2$getTickLineCoo.tick;
         var tickProps = _objectSpread(_objectSpread(_objectSpread(_objectSpread({
           textAnchor: textAnchor,
-          verticalAnchor: verticalAnchor
         }, axisProps), {}, {
           stroke: 'none',
           fill: stroke
         }, customTickProps), tickCoord), {}, {
           index: i,
           payload: entry,
-          visibleTicksCount: finalTicks.length,
-          tickFormatter: tickFormatter
         });
         return /*#__PURE__*/React.createElement(Layer, _extends({
           className: "recharts-cartesian-axis-tick",
